sdk/modelsrepository/Azure.Iot.ModelsRepository/src/Azure.Iot.ModelsRepository.csproj

```diff
@@ -0,0 +1,57 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <AssemblyTitle>Azure IoT Models Repository SDK</AssemblyTitle>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  </PropertyGroup>
+
+  <!-- Nuget properties -->
+  <PropertyGroup>
+    <PackageTags>IoT;ModelsRepository;Pnp;DigitalTwins$(PackageCommonTags)</PackageTags>
+    <Description>SDK for the Azure IoT Models Repository</Description>
+    <Version>1.0.0-preview.1</Version>
```

Comment:
Should this be `1.0.0-beta.1`?

Reply:
I think we use preview for .NET packages.

Reply:
I double checked, we used preview for ADT and appconfiguration used preview as well.